### PR TITLE
[feat] Save account creation status to DataStore on "Continue" butto…

### LINKED ISSUE #109: [feat] Save account information on creation
Currently, the account creation flow is just UI, and no actual account information is saved. When a user goes through the account creation flow, save their account information so that when the app is started up it takes the user directly to their account. 

Search for TODO(#109) to find where to change the code

### PR DESCRIPTION
**Pull request by Mystic-Nayy**
Resolves #109

### CODE REVIEW

**Review by Johnnysoltani — APPROVED** [review]

**Review by trbutler4 — CHANGES_REQUESTED** [review]
Please delete the local.properties file from git, and then add it to the .gitignore file. This shouldnt be committed to version control. My fault, not yours :)

File: wallet_app/app/src/main/java/com/example/walletapp/ui/onboarding/FinalizeAccountCreationScreen.kt [hunk 1]
```diff
@@ -269,15 +269,23 @@ fun AccountInfoView(onContinue: () -> Unit) {
...
+        // Navigate to the next screen
+        onContinue()
+    },
+    contentPadding = ButtonDefaults.ContentPadding,
+    shape = RoundedCornerShape(8.dp),
+    colors = ButtonDefaults.buttonColors(backgroundColor = Color("#EC796B".toColorInt()), contentColor = Color.White),
+    modifier = Modifier
+        .fillMaxWidth()
+        .height(49.dp
         ) {
```
Comment (trbutler4) [inline wallet_app/app/src/main/java/com/example/walletapp/ui/onboarding/FinalizeAccountCreationScreen.kt:289]: please fix this. 

image

**Review by trbutler4 — CHANGES_REQUESTED** [review]

**Comment by trbutler4** [comment]
@Mystic-Nayy please take a look at requested changes to get this done for ODHack

**Comment by Mystic-Nayy** [comment]
> @Mystic-Nayy please take a look at requested changes to get this done for ODHack

I have made the changes you requested for sir

**Comment by trbutler4** [comment]
> > @Mystic-Nayy please take a look at requested changes to get this done for ODHack
> 
> I have made the changes you requested for sir

there are conflicts, the app doesnt even run due to syntax errors. Use android studio

**Pull request closed without merging**